regexes/device/mobiles.yml

```diff
@@ -5570,10 +5854,16 @@ LG:
       model: 'Optimus Bright L-07C'
     - regex: '(?:LG-)?F800(?:S|K|L)?'
       model: 'V20'
-    - regex: 'LM-Q710(?:\(FGN\)|\.FGN)'
+    - regex: 'LM-Q710(?:\(FGN\)|\.FGN|FGN)'
```

Comment:
Wouldn't `'LM-Q710(?:\(FGN\)|\.?FGN)'` match the same?

Reply:
have no dot
```
Mozilla/5.0 Linux; Android 8.1.0; LM-Q710FGN AppleWebKit/537.36 KHTML, like Gecko Chrome/75.0.3770.89 Mobile Safari/537.36
```

Reply:
Sure. but the one with the dot is already there, so might be simpler to make the dot optional